### PR TITLE
Added samples for Gemini 1.5

### PR DESCRIPTION
**Pull request by meteatamel**
* Added some samples for Gemini 1.5. 
* They're all very similar samples with slight differences, that's why I'm doing it in a single PR. 
* Fixes b/336757387, b/336757211, b/336757534, b/336757434, b/336758092

### CODE REVIEW

**Comment by jskeet** [comment]
Reassigned to Amanda as a better samples reviewer - but feel free to assign it back, @amanda-tarafa

File: aiplatform/api/AIPlatform.Samples/VideoInputWithAudio.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        content.Parts.AddRange(new List<Part>()
+        {
+            new() {
+                Text = prompt
+            },
+            new() {
+                FileData = new() {
+                    MimeType = "video/mp4",
+                    FileUri = "gs://cloud-samples-data/generative-ai/video/pixel8.mp4"
+                }
```
Comment (amanda-tarafa) [inline aiplatform/api/AIPlatform.Samples/VideoInputWithAudio.cs:54]: nit: This braces should be on the next line.

**Review by amanda-tarafa — APPROVED** [review]
LGTM

But I noticed that for the anonymous types initialization you are using something like JSON syntax by keeping the openening bracket on the same line as `new`? Also using parentheses on all the `new`s.

In a separate PR, can you change that? So when you have things like:

```csharp
Field = new() {
    Innerfield = value
}
```

do this instead?

```csharp
Field = new
{
    Innerfield = value
}
```

On these samples and all the others. Thanks!

File: aiplatform/api/AIPlatform.Samples/VideoInputWithAudio.cs (hunk 1 above)
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/VideoInputWithAudio.cs:54]: @amanda-tarafa I'm handling this in a separate PR

**Pull request merged**